core/templates/cards.html

```diff
@@ -234,7 +234,7 @@ <h4 class="modal-title">Face Off</h4>
                 card.attr('onclick', 'flipNoScore(' + index + ')');
             }
         }
-        if (gameMode != easyGame) {
+        if (gameMode != easyGame && round != 4) {
             $("#countdown").hide();
```

Comment:
At the last round (5/5), you would select the card and then it would countdown as if there was another round to be played. But it shouldn't countdown because that is the last round, it should just show the results button.